Package/checkin

```diff
@@ -298,9 +299,15 @@ def main():
             enableduser = get_enabled_user()
         plist['EnabledUser'] = enableduser
         if 'last_run' in plist:
+            escrow_interval = int(pref('KeyEscrowInterval'))
+            logging.info('KeyEscrowInterval set to: {} hour(s)...'.format(
+                escrow_interval))
             now = datetime.datetime.now()
-            hour_ago = now - datetime.timedelta(hours=1)
+            hour_ago = now - datetime.timedelta(hours=escrow_interval)
```

Comment:
We should force this to an int. It's conceivable that someone will set it to a string. Possibly bail / default to 1 if it cannot be cast to an integer.